README.VLANs.md

```diff
@@ -0,0 +1,46 @@
+Using mstpd with Trunk Ports / VLANs
```

Comment:
this file could go as part of an initial commit set
it would be then added to `rpm.spec` when the entire `rpm.spec` is added